### PR TITLE
Added small note about shutdown

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/go-ipfs/pull/2180#issuecomment-171889567

License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: cmd/ipfs/daemon.go [hunk 1]
```diff
@@ -160,6 +160,7 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 		select {
 		case <-req.Context().Done():
 			fmt.Println("Received interrupt signal, shutting down...")
+			fmt.Println("(Hit C-c again to force-shutdown the daemon.)")
```
Comment (whyrusleeping) [inline cmd/ipfs/daemon.go]: maybe `ctrl+c` instead of `C-c` ?

Comment (RichardLitt) [inline cmd/ipfs/daemon.go]: Changed.

**Comment by noffle** [comment]
:+1:

**Comment by rht** [comment]
+1 LGTM

**Pull request merged**